packages/nouns-webapp/src/components/CurrentBid/index.tsx

```diff
@@ -15,16 +18,26 @@ type BidNa = typeof BID_N_A;
 
 const CurrentBid: React.FC<{ currentBid: BigNumber | BidNa; auctionEnded: boolean }> = props => {
   const { currentBid, auctionEnded } = props;
-
+  const isCool = useAppSelector(state => state.application.stateBackgroundColor) === "#d5d7e1";
   const titleContent = auctionEnded ? 'Winning bid' : 'Current bid';
 
   return (
-    <div className={classes.section}>
-      <h4>{titleContent}</h4>
-      <h2>
-        {currentBid === BID_N_A ? BID_N_A : <TruncatedAmount amount={currentBid && currentBid} />}
-      </h2>
-    </div>
+    <Container className={clsx(classes.wrapper, classes.container)}>
```

Comment:
let's try to keep `Container` components at top-level pages if not necessary - should be used the house the entire entire grid instead.